app/views/users/_user.html.erb

```diff
@@ -2,7 +2,7 @@
   <%= gravatar_for user, size: 50 %>
   <%= link_to user.username, user %>
 
-  <% if current_user.privilege == "admin" && !current_user?(user) %>
+  <% if current_user && current_user.privilege == "admin" && !current_user?(user) %>
```

Comment:
I'm not entirely sure why I needed to do this, but I did. Was getting an error otherwise with current_user being null